### PR TITLE
Vb veltson create branch, and kor3 site

### PR DESCRIPTION
**Pull request by veltsonbastien**
Since lots of changes happened, I think kor3's demo site still needs to get put up

### CODE REVIEW

File: src/pages/kor3.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        hMapScale={30}
+        xzMapScale={100}
+        floorColor="black"
+        night
+      >
+        <group position={[0, 0, 0]}>
+          {/*<mesh>*/}
+          {/*  <meshStandardMaterial attach="material" color="red" />*/}
+          {/*  <textBufferGeometry attach="geometry">Hello</textBufferGeometry>*/}
+          {/*</mesh>*/}
```
Comment (alex-shortt) [inline src/pages/kor3.tsx:79]: remove comment

File: src/pages/kor3.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        scenePos={[0, -20, 0]}
+        fogNear={0}
+        fogFar={150}
+        fogColor={"#000000"}
+        hMapScale={30}
+        xzMapScale={100}
+        floorColor="black"
+        night
+      >
+        <group position={[0, 0, 0]}>
```
Comment (alex-shortt) [inline src/pages/kor3.tsx:75]: don't need `position={[0, 0, 0]}`

File: src/pages/kor3.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+const LinkTree: NextPage = () => {
+  const material = useMemo(
+    () =>
+      new MeshStandardMaterial({
+        color: 0xffffff,
+        metalness: 0.2,
+        roughness: 0.1,
+      }),
+    []
+  );
```
Comment (alex-shortt) [inline src/pages/kor3.tsx:55]: pretty sure this is unused, feel free to delete

**Review by alex-shortt — CHANGES_REQUESTED** [review]

**Review by alex-shortt — APPROVED** [review]

File: src/pages/kor3.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        artwork={linkData}
+        socials={ARTIST.socialLinks}
+        name={ARTIST.name}
+        scenePos={[0, -20, 0]}
+        fogNear={0}
+        fogFar={150}
+        fogColor={"#000000"}
+        hMapScale={30}
+        xzMapScale={100}
+        floorColor="black"
```
Comment (alex-shortt) [inline src/pages/kor3.tsx:61]: these props are not offered by Gotham. do you mean OldGotham?

**Review by alex-shortt — CHANGES_REQUESTED** [review]
sorry for accepting, just realized the props are wrong

File: src/pages/kor3.tsx (hunk 4 above)
Comment (veltsonbastien) [reply src/pages/kor3.tsx:61]: Probably? I can change it to that

**Review by alex-shortt — APPROVED** [review]

**Pull request merged**